doc/input-output-reference/src/overview/group-system-availability-managers.tex

```diff
@@ -628,6 +637,16 @@ \subsubsection{Outputs}\label{outputs-9-008}
 
 HVAC,Average,Availability Manager Hybrid Ventilation Control Status {[]}
 
+HVAC,Average,Hybrid Ventilation Control HVAC System Opration Elapsed Time {[min]}
```

Comment:
@lgu1234 I wouldn't change the values from the status output, because users are accustomed to the 0,1,2 output.  We can leave the outputs as-is for now.  If there's a request later, we can add more.